### PR TITLE
select datasets without prondict for non-kaldi models

### PR DESCRIPTION
**Pull request by benfoley**
goes with https://github.com/CoEDL/elpis/pull/109

### CODE REVIEW

File: src/components/Model/CurrentModelName.js [hunk 1]
```diff
@@ -23,8 +23,12 @@ class CurrentModelName extends Component {
             <Message color='olive'>
                     {t('dataset.common.currentDatasetLabel') + datasetName}
                     <br />
+                {pronDictName &&
+                    <>
```
Comment (nicklambourne) [inline src/components/Model/CurrentModelName.js:27]: What does this empty tag do?

File: src/components/Model/NewForm.js [hunk 2]
```diff
@@ -98,23 +129,23 @@ class NewForm extends Component {
 
 const mapStateToProps = state => {
     return {
+        engine: state.sideNav.engine,
```
Comment (nicklambourne) [inline src/components/Model/NewForm.js:132]: Why is this info stored in sideNav?

File: src/components/Model/CurrentModelName.js (hunk 1 above)
Comment (benfoley) [reply src/components/Model/CurrentModelName.js:27]: This is shorthand for a "fragment", it wraps sibling expressions/components without adding extra DOM elements. 

"Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>"

The expression that is being wrapped here displays the prondict name and a line break. For example:
```
<p>name</p>
<br />
```

For this to be valid JSX, we require a parent element for the `p` and `br` sibling elements. We used to have to do this:
```
<div>
<p>name</p>
<br />
</div>
```

but that adds extra DOM unnecessarily and the amount of times it happens, the DOM gets littered with divs.

Fragments look like this
```
<React.Fragment>
<p>name</p>
<br />
</React.Fragment>
```

and shorthand version replaces `<React.Fragment>` with `<>`

File: src/components/Model/NewForm.js (hunk 2 above)
Comment (benfoley) [reply src/components/Model/NewForm.js:132]: No idea, I think Nic put the engines redux stuff in the sidenav reducer. Would be clearer if there was an engineReducer or appReducer. it works for now, I have a reminder to clean it up later.

**Pull request merged**